css/index.css

```diff
@@ -1,3 +1,504 @@
+	0% {
+		transform: scale(1);
+	}
+
+	100% {
+		transform: scale(1.25);
+	}
+}
+
+nav a {
+	color: black;
+	text-decoration: none;
+	font-weight: bold;
+	padding: 17px 0;
+}
+
+nav a:hover {
+  color: yellow;
+  transition: color 0.5s ease-out;
+  animation: wobble 0.4s linear 1;
+}
+
+@keyframes wobble {
+  0%, 50%, 100% {
+    transform: translate(0,0);
+  }
+
+  25% {
+    transform: translate(4px);
+  }
+
+  75% {
+    transform: translate(-4px);
+  }
+}
+
+nav a:active {
+  color: red;
+}
+
+/* Innovation */
+.innovation {
+	justify-content: space-between;
+	align-items: center;
+	padding: 60px 3px;
+	margin-top: 42px;
+}
+
+.innovation h1 {
+	text-align: center;
+  font-size: 86px;
+  letter-spacing: 7px;
+}
+
+.title {
+	text-align: center;
+}
+
+.title a {
+	margin: 0 100px;
+  padding: 11px 0;
+  display: block;
+  text-decoration: none;
+  color: inherit;
+  font-weight: bold;
+  border: 2px solid black;
+}
+
+.innovation a:hover {
+  animation: swing 0.5s linear .12s alternate;
+  background-color: black;
+  color: white;
+  transition: background-color 0.5s linear .12s, color 0.5s linear .12s;
+}
+
+@keyframes swing {
+  0%, 50%, 100% {
+    transform: rotate(0);
+  }
+
+  25% {
+    transform: rotate(15deg);
+  }
+
+  75% {
+    transform: rotate(-15deg);
+  }
+}
+
+/* Information */
+.box {
+	align-items: center;
+}
+
+.top {
+  padding: 40px 0px;
+  width: 50%;
+}
+
+.bottom {
+  padding: 40px 0;
+  width: 34%;
+}
+
+.middle-img {
+  width: 100%;
+}
+
+.product {
+	padding: 0 10px;
+	margin: 0 5px;
+}
+
+/* Footer */
+.contact {
+  padding: 20px 0;
+  line-height: 2.5;
+}
+
+.address {
+  line-height: 1.3;
+}
+
+.copyright {
+  text-align:center;
+  padding-bottom: 50px;
+  color: black;
+  font-weight: bold;
+}
+
+
+/* Service Page */
+.serv p, .pins p, .facts p {
+	line-height: 1.2;
+}
+
+/* service */
+.serv {
+	margin-top: 75px;
+	padding-bottom: 40px; 
+}
+
+.serv div {
+	margin-top: 20px;
+}
+
+.serv h1 {
+	font-size: 26px;
+}
+
+.serv img {
+	width: 100%;
+}
+
+/* Pins */
+.pins {
+	flex-wrap: wrap;
+	justify-content: space-between;
+	margin-top: 40px;
+	padding-bottom: 15px;
+}
+
+.pin {
+	width: 47%;
+	height: auto;
+	padding: 25px 41px;
+	margin: 26px 0;
+	border: 2px solid black;
+	background-color: #eeebeb86;
+}
+
+.button {
+	border: 2px solid black;
+	border-radius: 5px;
+	width: 40%;
+	padding: 9px;
+	margin-top: 20px; 
+	text-align: center;
+	background-color: white;
+	color: black;
+	transition: background-color 0.5s, color 0.5s;
+}
+
+.pin a {
+	text-decoration: none;	
+}
+
+.button:hover {
+	background-color: black;
+	color: white;
+}
+
+/* Facts */
+.facts {
+	margin-top: 40px; 
+}
+
+.fact p {
+	padding: 3px 90px 16px 0; 
+}
+
+.fact ul li {
+	list-style-type: square;
+	list-style-position: inside; 
+	padding: 9px 0;
+}
+
+/* Footer */
+.space {
+	margin-top: 50px; 
+	padding-bottom: 20px; 
+}
+
+
+/* Media queries */
+/* Tablet version */
+@media (max-width: 800px) {
+  html, body {
+    max-width: 720px;
```

Comment:
Instead of adjusting .container to new fixed widths with media queries, you can actually just set it to a % based width and also use that in combination with a max-width property. So, for example: 
.container {
width: 100%;
max-width: 880px;
margin: 0 auto;
}
That way, at screen sizes < 880px, it takes up the whole screen. At > 880px, it takes up only 880px and auto centers itself.

With this method, you won't have that horizontal scroll bar

Reply:
Nice job! It looks great. Just see above for one note on making the design 100% responsive so that you never get a horizontal scroll bar at any screen width (for instance, currently there's a scrollbar between the 880px .container width and when the media query changes it at 800px).